content/docs/command-reference/get-url.md

```diff
@@ -9,7 +9,7 @@ Download a file or directory from a supported URL (for example `s3://`,
 ## Synopsis
 
 ```usage
-usage: dvc get-url [-h] [-q | -v] [-j <number>] url [out]
+usage: dvc get-url [-h] [-q | -v] url [out]
```

Comment:
Oops @isidentical I'm seeing lots of `-j` related changes here. Maybe this got contaminated from another of your docs branches?

Reply:
This file (content/docs/command-reference/get-url.md), content/docs/command-reference/get.md, and content/docs/command-reference/import.md to be precise.